src/Wrapper.js

```diff
@@ -307,6 +315,7 @@ const Main = styled.div`
   display: flex;
   flex-direction: column;
   height: 100vh;
+  min-width: 320px;
```

Comment:
❤️ with this change none of the apps need to set the `min-width` no?

Reply:
> ❤️ with this change none of the apps need to set the `min-width` no?

Yes :-)

> Even though, I think I'd encourage it in case of headless apps.

Yes I think we should keep it as a good practice.